### PR TITLE
Fix user switch when rating appilcation

### PR DESCRIPTION
**Pull request by ShayGoldman**
- Added updateAdmin @ /users/updateAdmin
- Changed call from application review to updateAdmin rather then update

### CODE REVIEW

**Comment by aviadhahami** [comment]
Tested ?

**Comment by ShayGoldman** [comment]
Look in trello

**Comment by aviadhahami** [comment]
Post link please , @ShayGoldman

File: public/modules/dashboard/controllers/applications-review.client.controller.js [hunk 1]
```diff
@@ -31,34 +29,48 @@ angular.module('dashboard').controller('applicationsReviewController', ['$scope'
             $scope.selectedApplication = undefined;
         };
 
+        function updateUser(user) {
+            var deferred = $q.defer();
```
Comment (aviadhahami) [inline public/modules/dashboard/controllers/applications-review.client.controller.js]: No need to create new promise, $http is a promise itself

File: app/controllers/users/users.profile.server.controller.js [hunk 2]
```diff
@@ -10,6 +10,36 @@ var _ = require('lodash'),
 	User = mongoose.model('User'),
 	rolesString = 'volunteer,participant,manager,admin';
 
+exports.updateAdmin = function(req, res) {
+	// Init Variables
+	var user = req.user;
+
+	// For security measurement we remove the roles from the req.body object
+	delete req.body.roles;
```
Comment (aviadhahami) [inline app/controllers/users/users.profile.server.controller.js:18]: How do you check that the user accessed this URI is an admin?

File: app/controllers/users/users.profile.server.controller.js [hunk 3]
```diff
@@ -10,6 +10,36 @@ var _ = require('lodash'),
...
+
+	if (user) {
+		// Merge existing user
+		user = _.extend(user, req.body);
```
Comment (aviadhahami) [inline app/controllers/users/users.profile.server.controller.js]: You are merging the admin user `req.user` with the request body ?

**Comment by aviadhahami** [comment]
@ShayGoldman Please take a look at the things I've changed

**Pull request merged**